Sources/TarkaUI/Components/Base Components/RadioButton/TUIRadioButton.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  TUIRadioButton.swift
+//
+//
+//  Created by MAHESHWARAN on 02/02/24.
+//
+
+import SwiftUI
+
+public struct TUIRadioButton: View {
+  
+  private var inputItem: InputItem
+  
+  public init() {
+    self.inputItem = .init()
+  }
+  
+  public var body: some View {
+    Circle()
+      .stroke(inputItem.color, lineWidth: 1.5)
+      .frame(width: 24, height: 24)
+      .overlay(content: backgroundColor)
```

Comment:
pls add accessibility id for this main view

Reply:
@gopinathcholan Updated